model/tensorflow/dffml_model_tensorflow/dnnc.py

```diff
@@ -139,10 +139,10 @@ def model(self):
 
 @config
 class DNNClassifierModelConfig:
-    classification: Feature = field(
-        "Feature name holding classification value"
+    predict: Feature = field(
```

Comment:
We should really do the `classification` -> `predict` in a seperate pull request. It's currentl in both `sql_op` and this. And it's really it's own thing.

Reply:
Yeah,sorry my bad,created [PR](https://github.com/intel/dffml/pull/340)

Reply:
Merged, thanks!